.github/workflows/npm-release.yml

```
@@ -0,0 +1,93 @@
+name: npm-release
+
+on:
```

yamllint (1.35.1)

[warning] truthy value should be one of [false, true]

(truthy)